src/components/movie-card/movie-card.jsx

```diff
@@ -1,12 +1,28 @@
-import React from "react";
+import React from 'react';
+import PropTypes from 'prop-types';
 
 export class MovieCard extends React.Component {
   render() {
     const { movieData, onMovieClick } = this.props;
-    
+
```

Comment:
This component doesn't require state. So you might benefit from writing it as a functional component similar to the Login Component.